dist/tempManager.js

```diff
@@ -0,0 +1,61 @@
+class TempManager {
+    constructor() {
+        this.cityData = []
+    }
+
+    async getDataFromDB() {
```

Comment:
`getCities()`
once you start dealing with more routes and more data you need to be more specific about your methods so it is easier for you to follow and for others to read